README.md

```diff
@@ -956,7 +968,20 @@ A simple use-case could be as follows:
     2. retrieving the token value from the response
     3. inserting the token value and user ID into the edition's target URL
     4. returning a 303 redirect with the modified edition URI as the target
-3. The reader's browser redirects to the edition and the edition uses the token to authenticate.
+3. The reader's browser redirects to the edition and the edition uses the token to authenticate, bypassing the login requirement.
+
+As a second example, if your edition URL is `http://hosted.edition.domain/path/to/edition/index.html` then the token above could be used by directing the user to the destination `http://hosted.edition.domain/path/to/edition/index.html?yuduAuthId=uniqueUserIdentification&yuduAuthToken=0123456789abcdefghijklmnopqrstu&ugcKey=12345`.
+
+A simple use-case could be as follows:
+
+1. A reader clicks on a link on your webpage indicating they wish to view an edition.
+2. Your server looks up the Reader ID for that user (and probably should cache this in your database so this step isn't necessary in future, making it faster for the end-user).
+3. Your server reacts to that request by:
+    1. sending a request to this API for a token for a specific user ID (this can be anything as long as it's unique, but since you've just looked up the Reader ID, feel free to use that).
+    2. retrieving the token value from the response
+    3. inserting the user ID, token value, Reader ID into the edition's target URL with the query parameter names shown above
```

Comment:
*inserting the user ID, token value _and_ Reader ID... ? Sounds more natural.

Use the same wording as here at point 3.3 in the first example (excluding 'Reader ID')?